### PR TITLE
Refactor(examples): Login

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue
https://github.com/DivanteLtd/storefront-ui/issues/903

# Scope of work
Reafactor Login example page

# Screenshots of visual design
image

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed

If applicable:

- [] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))
 - [] I accept [Individual Contributor License Agreement](https://github.com/DivanteLtd/next/blob/master/CLA.md)

### CODE REVIEW

**Comment by pspaczek** [comment]
the first part of CR, visual changes
1. Please check colors, and spacing
PR
Zrzut ekranu 2020-04-1 o 10 20 47
Design
Zrzut ekranu 2020-04-1 o 10 20 00
2. Like up spacing and line-height
PR
Zrzut ekranu 2020-04-1 o 10 24 48
Design
Zrzut ekranu 2020-04-1 o 10 24 41
3. Please check button's actions and adjust it to :)
Zrzut ekranu 2020-04-1 o 10 26 14
4. IMO on mobile we need just close button like on [master storybook](https://storybook.storefrontui.io/?path=/story/pages-login-sign-in--common)
Zrzut ekranu 2020-04-1 o 10 28 14

**Comment by pspaczek** [comment]
ok... IMO this is a part of MyAccount, we should remove it from Login/Register example, safe this code for the feature on your local files
Zrzut ekranu 2020-04-1 o 10 57 13

File: packages/vue/src/examples/pages/login/Login.vue [hunk 1]
```diff
@@ -1,89 +1,125 @@
 <template>
   <div>
```
Comment (pspaczek) [inline packages/vue/src/examples/pages/login/Login.vue:2]: when you use just SfModal you can remove this div

File: packages/vue/src/examples/pages/login/Login.vue [hunk 2]
```diff
@@ -116,35 +191,105 @@ export default {
...
+      this.isMyAccount = false;
+      if (btn === "signIn") {
+        this.isSignIn = true;
+        this.isLogin = false;
+      } else if (btn === "join") {
+        this.isLogin = true;
+        this.isSignIn = false;
+      }
+    },
+    onScreenChange() {
```
Comment (pspaczek) [inline packages/vue/src/examples/pages/login/Login.vue:206]: you should use computed for it 👍

File: packages/vue/src/examples/pages/login/Login.vue [hunk 3]
```diff
@@ -1,89 +1,125 @@
...
+          >
+        </div>
+        <div v-else-if="isLogin" key="log-in" class="login__form__container">
+          <div class="login__form">
             <SfInput
               v-model="email"
               name="email"
               label="Your email"
-              class="form__element"
+              class="login__form__element"
```
[the remark this answers is not in the document]
Comment (AdamPawlinski) [reply packages/vue/src/examples/pages/login/Login.vue]: ok, view removed and styles adjusted

**Pull request merged**